--- a/config.def.h	2024-05-16 21:00:16.932060677 +0200
+++ b/config.def.h	2024-05-17 22:49:07.412917962 +0200
@@ -98,46 +98,49 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-	"gray90", /* default foreground colour */
-	"black", /* default background colour */
-};
 
+  /* 8 normal colors */
+  [0] = "#15161e", /* black   */
+  [1] = "#f7768e", /* red     */
+  [2] = "#9ece6a", /* green   */
+  [3] = "#e0af68", /* yellow  */
+  [4] = "#7aa2f7", /* blue    */
+  [5] = "#bb9af7", /* magenta */
+  [6] = "#7dcfff", /* cyan    */
+  [7] = "#a9b1d6", /* white   */
+
+  /* 8 bright colors */
+  [8]  = "#414868", /* black   */
+  [9]  = "#f7768e", /* red     */
+  [10] = "#9ece6a", /* green   */
+  [11] = "#e0af68", /* yellow  */
+  [12] = "#7aa2f7", /* blue    */
+  [13] = "#bb9af7", /* magenta */
+  [14] = "#7dcfff", /* cyan    */
+  [15] = "#c0caf5", /* white   */
+
+  /* special colors */
+  [256] = "#111927", /* background */
+  [257] = "#c0caf5", /* foreground */
+};
 
 /*
  * Default colors (colorname index)
- * foreground, background, cursor, reverse cursor
+ * foreground, background, cursor
  */
-unsigned int defaultfg = 258;
-unsigned int defaultbg = 259;
-unsigned int defaultcs = 256;
+unsigned int defaultfg = 257;
+unsigned int defaultbg = 256;
+unsigned int defaultcs = 257;
 static unsigned int defaultrcs = 257;
 
 /*
+ * Colors used, when the specific fg == defaultfg. So in reverse mode this
+ * will reverse too. Another logic would only make the simple feature too
+ * complex.
+ */
+static unsigned int defaultitalic = 7;
+static unsigned int defaultunderline = 7;
+/*
  * Default shape of cursor
  * 2: Block ("█")
  * 4: Underline ("_")
